### ISSUE TITLE
Certificate verify failed

### ISSUE THREAD

**Issue by Mattia2700** [nkanand4/google-photos-for-kodi#2]
When I click for insert the account it givers me an error; looking in the logs shows me that there is a CERTIFICATE_VERIFY_FAILED, idk maybe is because it is an old script

**Comment by nkanand4** [comment]
Yes, it's very old, @Mattia2700 and I am not sure if it makes sense to work on it now because they used to go through picasa API which got obsoleted.

**Comment by Mattia2700** [comment]
Oh yes, I understand, gg for your work btw 💯
